src/gwt/src/org/rstudio/studio/client/workbench/ui/PaneManager.java

```diff
@@ -925,128 +928,37 @@ else if (window.equals(getConsoleLogicalWindow()))
-               return;
-
-            // The logic here is more complex than for the right panel because there may be
-            // multiple widgets and the animation needs to occur across all the widgets rather
-            // than in each specified widget.
-            size = (1 - progress) * leftStartSum +
-               progress * leftEndSum;
-
-            final double currentSize = panel_.getLeftSize();
-            if (currentSize > size) // we are shrinking
-            {
-               double difference = currentSize - size;
-               for (int i = 0; i < leftStart.size(); i++)
-               {
-                  double widgetSize = panel_.getWidgetSize(leftList_.get(i));
-                  if (widgetSize > 0)
-                  {
-                     if (widgetSize > difference)
-                     {
-                        panel_.setWidgetSize(leftList_.get(i), widgetSize - difference);
-                        break;
-                     }
-                     else
-                     {
-                        panel_.setWidgetSize(leftList_.get(i), 0.0);
-                        difference -= widgetSize;
-                        if (difference <= 0.0)
-                           break;
-                     }
-                  }
-               }
-            }
-            else if (currentSize < size)// we are growing
-            {
-               // iterate backwards so the left most widget is shown first
-               for (int i = leftStart.size() - 1; i >= 0; i--)
-               {
-                  final double widgetSize = panel_.getWidgetSize(leftList_.get(i));
-                  // If the widget is bigger than the size, calculate the size of the display
-                  if (widgetSize < leftEnd.get(i))
-                  {
-                     if (size > leftEnd.get(i))
-                     {
-                        panel_.setWidgetSize(leftList_.get(i), leftEnd.get(i));
-                        size -= leftEnd.get(i);
-                     }
-                     else
-                     {
-                        panel_.setWidgetSize(leftList_.get(i), size);
-                        break;
-                     }
-                  }
-                  else
-                  {
-                     size -= widgetSize;
-                     if (size <= 0.0)
-                        break;
-                  }
-               }
-            }
-         }
-
-         @Override
-         protected void onStart()
-         {
-            isAnimating_ = true;
-            super.onStart();
-         }
-
-         @Override
-         protected void onComplete()
-         {
-            isAnimating_ = false;
             panel_.onSplitterResized(new SplitterResizedEvent());
-            super.onComplete();
             if (afterComplete != null)
                afterComplete.execute();
          }
-      };
+         
+         public void onLayout(Layer layer, double progress)
```

Comment:
Is this empty method needed?

Reply:
Yes because it's part of the `AnimationCallback` interface. It gets called at each step of animation but we don't need to do anything special for that here.